tests/runtest.sh

```diff
@@ -572,7 +572,7 @@ function finish_test {
 
     if [ "$showTime" == "ON" ]; then
         testEndTime=$(date +%s)
-        testRunningTime=$(echo "$testEndTime - ${testStartTimes[$nextProcessIndex]}" | bc)
+        let "testRunningTime = ${testEndTime} - ${testStartTimes[$nextProcessIndex]}"
```

Comment:
Why `let` instead of classic `$((expression))`:
`testRunningTime="$((${testEndTime} - ${testStartTimes[$nextProcessIndex]}))"`